libr/core/anal.c

```diff
@@ -310,7 +539,8 @@ static char *r_core_anal_graph_label(RCore *core, RAnalBlock *bb, int opts) {
 						memcpy (cmdstr + idx, "\\n", 3);
 					else if (is_html)
 						memcpy (cmdstr + idx, "<br />", 7);
-					else memcpy (cmdstr + idx, "\\l", 3);
+					else
```

Comment:
brackets on this one, it's scary to have one line in an if/else then more after it.